yarn-project/foundation/src/config/env_var.ts

```diff
@@ -126,6 +126,7 @@ export type EnvVar =
   | 'VALIDATOR_ATTESTATIONS_POOLING_INTERVAL_MS'
   | 'PROVER_NODE_DISABLE_AUTOMATIC_PROVING'
   | 'PROVER_NODE_MAX_PENDING_JOBS'
+  | 'PROVER_NODE_EPOCH_SIZE'
```

Comment:
Haven't yet seen how this is used but should this not come from the contract?

Reply:
Yes. I'm letting the prover choose its own epoch size for the time being.